scripts/validate-go.sh

```diff
@@ -19,7 +19,6 @@ exit_code=0
 
 if ! hash gometalinter 2>/dev/null ; then
   go get -u github.com/alecthomas/gometalinter
-  gometalinter --install
```

Comment:
I think we need to keep this as first we 1) install gometalinter itself and 2) install the linters themselves.